webapp/src/theme.ts

```diff
@@ -128,12 +141,26 @@ export function setTheme(theme: Theme | null): Theme {
     return consolidatedTheme
 }
 
+function setActiveThemeName(consolidatedTheme: Theme, theme: Theme | null) {
+    if (theme === null) {
+        activeThemeName = systemThemeName
+    } else if (_.isEqual(consolidatedTheme, darkTheme)) {
```

Comment:
Maybe is a bit overkill to add a whole new library to this tiny functionality.

Reply:
Continuing using this as its simpler way, but importing only required function instead of the whole package.